docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+# Docker Compose file for Archivy
+version: "3.8"
+services:
+  archivy:
+    image: harshavardhanj/archivy
+    ports:
+      - "5000:5000"
+    volumes:
```

Comment:
Could you explain what the `volumes` are here?

Reply:
I've done it in the README just now. The `volumes` option lets you attach a volume to the container. That way, if and when the container is terminated, the data isn't erased/deleted. That way, you can stop/terminate a container without affecting the persistent data, and start it back up with the same volume. It's a way to keep data persistent on containers. Here, a directory `data` is created in the current working directory(`./data`) and bind-mounted to the `/usr/src/app/data` directory on the container(which happens to be where the application data will be stored).